test/e2e/velero_utils.go

```diff
@@ -269,22 +269,25 @@ func veleroBackupLogs(ctx context.Context, veleroCLI string, veleroNamespace str
 	return nil
 }
 
-func veleroRestoreLogs(ctx context.Context, veleroCLI string, veleroNamespace string, restoreName string) error {
-	describeCmd := exec.CommandContext(ctx, veleroCLI, "--namespace", veleroNamespace, "restore", "describe", restoreName)
-	describeCmd.Stdout = os.Stdout
-	describeCmd.Stderr = os.Stderr
-	err := describeCmd.Run()
-	if err != nil {
-		return err
+func runDebug(veleroCLI, veleroNamespace, backup, restore string) {
```

Comment:
IMO we should keep it consistent with `veleroRestoreLogs` to pass ctx as the first param to func `runDebug`